### ISSUE TITLE
Point Cloud Learning

### ISSUE THREAD

**Issue by asiamanman** [rusty1s/pytorch_geometric#2717]
Do you know how to train the point cloud data measured by LiDAR converted into a PCD file as training data?

**Comment by rusty1s** [comment]
You mean something like [this](http://www.cvlibs.net/datasets/kitti/)?

**Comment by asiamanman** [comment]
I think the data (ModelNet10) used in pointnet2_classification.py is the data of the cad system with the extension .off.
I want to train data with a .pcd extension.
In that case, what part of the source code of pointnet2_classification.py should be changed?

**Comment by rusty1s** [comment]
You can use the original `ModelNet` dataset and pass in the `SamplePoints` transform.

**Comment by asiamanman** [comment]
OK, thank you.
I think that the output label is calculated by pred = model (data) .max (1) [1] in the function test in pointnet2_classfication. How is this result calculated?

**Comment by rusty1s** [comment]
`model(data)` returns the output probabilities for each class. We simply take the class with the highest probability as prediction, e.g., via `max(dim=-1)[1]` or `argmax(dim=-1)`.

**Comment by asiamanman** [comment]
OK, Thank you
